_cucumber/lib/pages/primary_nav_menu.rb

```diff
@@ -0,0 +1,28 @@
+class PrimaryNav < Base
+
+  PRIMARY_NAV_BAR = '.primary-nav'
+
+  def links(link)
+    page.within(PRIMARY_NAV_BAR) do
+      case link
+        when 'Solutions'
+          page.has_link?('Solutions', href: BASE_URL+'/solutions/')
```

Comment:
Not an issue with this PR, but @paulrobinson shouldn't these primary nav links be relative? Minor point though.